docs/description_output_dataframes.rst

```diff
@@ -0,0 +1,143 @@
+Interpretations
+===============
+
+Possible interpretations are:
+ * Informele stratigrafie
+ * Formele stratigrafie
+ * Lithologische beschrijvingen
+ * Gecodeerde lithologie
+ * Hydrogeologische stratigrafie
+ * Informele hydrogeologische stratigrafie
+ * Quartaire stratigrafie
+ * Geotechnische coderingen
+
+Below the desired attributes for each interpretation as 
+item /newline/ new_name, data_type, example
+The new_name column represents the headers of the final dataframe.
+Some elements can occur more than once, e.g. 'bijmenging' in 'Gecodeerde
+lithologie' or 'Geotechnische coderingen'. All occurrences should be included 
+as new records in the final dataframe.
+
+  .. csv-table:: Informele stratigrafie
+    :header-rows: 1
+
+    source,new_name,data_type,example
+    url,permkey,full_url,https://.../2001-186513.xml
+    /ns3:dov-schema/interpretaties/informelestratigrafie/laag/van,van,float,0.00
+    /ns3:dov-schema/interpretaties/informelestratigrafie/laag/tot,tot,float,1.74
+    /ns3:dov-schema/interpretaties/informelestratigrafie/laag/beschrijving,beschrijving,string,Quartair
+
+|
+
+ .. csv-table:: Formele stratigrafie
+    :header-rows: 1
+
+    source,new_name,data_type,example
+    url,permkey,full_url,https://.../2001-186513.xml
```

Comment:
Is this the permkey of the interpretation or from the borehole? I assume the key of the interpretation. Shouldn't we need to provide a permkey for the borehole as well? @Roel